### ISSUE TITLE
inventory issue

### ISSUE THREAD

**Issue by juliacmcf** [diamm/diamm#435]
Is there any way we could identify an instance of a piece as a fragment? If not we have to create a second composition but note in the composition identity that it's a fragment. In the original database we had a field to show what a piece looked like in the original source, and that was linked to the composition. In the new version we have to link a piece to its composition, but have no way to indicate the state or completeness of the iteration in a MS inventory.

**Comment by juliacmcf** [comment]
Another related problem: we have pieces which are essentially the same musically, but have different texts. If we have in the composition two texts ... 'text aaa = text bbb' this denotes the same piece, but not which version is in the MS (because the inventory only lists the composition title, not the title in the MS). We're looking at Sadler at the moment, where several pieces have concordances elsewhere, but variant texts in Sadler, and the Sadler variants are crucial to the provenance of the books. If we separate them into separate pieces then we can no longer see the concordances. Overall then, the title in the Manuscript is always the more important piece of information to show in the inventory. This is particularly important where we don't have pictures, but even if we do have pictures there's nothing in the composition information showing in the inventory that would warn people that the source they're looking at has a variant text.

**Comment by ahankinson** [comment]
There are notes fields in the Item record. Would that work for recording the state of the composition in a source?

**Comment by juliacmcf** [comment]
Yes, but I’m not sure the notes are showing up correctly on the live site...

**Comment by ahankinson** [comment]
Do you have an example of one that is not?
